VideoLocker/src/main/java/org/edx/mobile/view/adapters/CourseDiscussionResponsesAdapter.java

```diff
@@ -292,8 +293,21 @@ public void onException(Exception ex) {
         holder.socialLayoutViewHolder.threadFollowContainer.setVisibility(View.INVISIBLE);
 
         if (comment.isEndorsed()) {
+            DiscussionThread.ThreadType threadType = discussionThread.getType();
+            @StringRes int stringRes;
```

Comment:
This variable name could be changed to something more descriptive, such as `attributionStringRes`.